pkg/epp/plugins/plugins.go

```diff
@@ -21,9 +21,11 @@ package plugins
 type Plugin interface {
 	// Type returns the type of the plugin.
 	Type() string
+	// Name returns the name of the plugin.
```

Comment:
nit: 
```suggestion
	// Name returns the name of the plugin instance.
```

otherwise, it's a bit hard to understand what's the difference between type and name.